### PR TITLE
Fix margin around the button on about:safebrowsing & Add browserButton_subtleItem fixing regressions

### LINKED ISSUE #8945: button styling on about:safebrowsing
## Test plan

https://github.com/brave/browser-laptop/pull/9026#issue-231104652

----

**Describe the issue you encountered:**
https://github.com/brave/browser-laptop/pull/8923#issuecomment-302490135
> side note @luixxiul could you please fix this one, I saw it while testing this PR. Button is jumping around :smile: 
> 
> 

- Brave Version (revision SHA): https://github.com/brave/browser-laptop/commit/77140b9b083090a0417032f52cbdb7725c964783

- Extra QA steps:
    1.
    2.
    3.

- Any related issues:

### LINKED ISSUE #9072: Add browserButton_subtleItem to BrowserButton component
## Test plan

https://github.com/brave/browser-laptop/pull/9026#issue-231104652

----

**Describe the issue you encountered:** Add browserButton_subtleItem to BrowserButton component.

- Extra QA steps:
    1.
    2.
    3.

- Any related issues: #9071

### PR DESCRIPTION
**Pull request by luixxiul**
On the 1st commit:
Fix margin around the button on about:safebrowsing

On the 2nd commit:
Add browserButton_subtleItem fixing regressions
- Add lines in the same way as LESS files
- Add comments for refenrence (We could remove them after refactoring work)
- Fix regressions on browserButton_default
  - Add globalStyles.button.color to replace it with globalStyles.button.default.color
- Fix a typo

----

Closes #8945
Closes #9072
Addresses #9025
Addresses #9071

Auditors:

**Test Plan 1:**
1. Take a screenshot of the buttons section on about:styles before testing the PR for later use
2. Open about:styles
3. Click "buttons"
4. Take another screenshot of the section
5. Compare the two screenshots and make sure they look same

**Test Plan 2:**
1. Open about:styles
2. Toggle developer tools
3. Make sure button's padding is set to '5px 16px'

**Test Plan 3:**
1. Open downloadme.org
2. Click "Advanced" button
3. Make sure the button is not pushed down
4. Click "Hide advanced" button
5. Make sure the button does not move

----

Submitter Checklist:

- [x] Submitted a [ticket](https://github.com/brave/browser-laptop/issues) for my issue if one did not already exist.
- [x] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- [x] Ran `git rebase -i` to squash commits (if needed).

Reviewer Checklist:

Tests

- [ ] Adequate test coverage exists to prevent regressions
- [ ] Tests should be independent and work correctly when run individually or as a suite [ref](https://github.com/brave/browser-laptop/wiki/Code-Guidelines#test-dependencies)
- [ ] New files have MPL2 license header

### CODE REVIEW

File: js/about/safebrowsing.js [hunk 1]
```diff
@@ -26,18 +28,48 @@ class SafebrowsingPage extends React.Component {
...
+      <section>
         {this.state.advanced
-          ? <div>
-            <Button l10nId='safebrowsingErrorHideAdvanced' className='subtleButton' onClick={this.onAdvancedToggle.bind(this)} />
-            <div className='subtleText'><p data-l10n-id='safebrowsingErrorInfo' /><br /><p data-l10n-id='safebrowsingErrorBypass' /></div></div>
-          : <Button l10nId='safebrowsingErrorAdvanced' className='subtleButton' onClick={this.onAdvancedToggle.bind(this)} />}
-      </div>
+          ? <section>
+            <div className='buttons'>
+              <Button className='subtleButton'
```
Comment (cezaraugusto) [inline js/about/safebrowsing.js]: could we take advantage of this PR and add subtleButton to `<BrowserButton>` component?

Comment (luixxiul) [inline js/about/safebrowsing.js]: ah right, I've totally forgot about that component :-p

I'll replace it with that. Thanks for heads up.

**Comment by luixxiul** [comment]
on the 2nd commit I fixed several regressions introduced with #8568 for 0.15.3, which seems to me a not blocker. I think it's fine to wait for 0.16 series.

File: app/renderer/components/common/browserButton.js [hunk 2]
```diff
@@ -68,49 +67,71 @@ const styles = StyleSheet.create({
...
+    // cf: https://github.com/brave/browser-laptop/blob/548e11b1c889332fadb379237555625ad2a3c845/less/button.less#L94-L95
+    paddingTop: '5px',
+    paddingBottom: '5px',
+
+    // cf: https://github.com/brave/browser-laptop/blob/548e11b1c889332fadb379237555625ad2a3c845/less/button.less#L105-L106
+    paddingRight: '16px',
+    paddingLeft: '16px',
+
+    // cf: https://github.com/brave/browser-laptop/blob/548e11b1c889332fadb379237555625ad2a3c845/less/button.less#L98
+    minWidth: `calc(${globalStyles.spacing.defaultFontSize} * 6)`, // issue #6384
```
Comment (cezaraugusto) [inline app/renderer/components/common/browserButton.js]: nice catch

File: app/renderer/components/common/browserButton.js [hunk 3]
```diff
@@ -127,12 +149,31 @@ const styles = StyleSheet.create({
...
     fontSize: '13px',
     marginRight: '10px',
     padding: '2px 15px',
     textTransform: 'capitalize',
     width: 'auto'
+  },
+
+  browserButton_subtleItem: {
+    // cf: https://github.com/brave/browser-laptop/blob/548e11b1c889332fadb379237555625ad2a3c845/less/button.less#L151-L152
+    background: '#ccc',
```
Comment (cezaraugusto) [inline app/renderer/components/common/browserButton.js]: this should belong to global styles

Comment (luixxiul) [inline app/renderer/components/common/browserButton.js]: Fixed with https://github.com/brave/browser-laptop/commit/03a9abd783581d19fc11426bb7628c3452b56876

**Comment by luixxiul** [comment]
@cezaraugusto thanks for review.

**Pull request merged**